python_modules/dagster/dagster/_core/definitions/asset_graph.py

```diff
@@ -105,14 +107,16 @@ def from_assets(all_assets: Sequence[Union[AssetsDefinition, SourceAsset]]) -> "
 
             else:
                 check.failed(f"Expected SourceAsset or AssetsDefinition, got {type(asset)}")
-        return AssetGraph(
+        return InternalAssetGraph(
```

Comment:
this is a bit weird, as the ExternalAssetGraph contains its own static constructor, but the internal asset graph is constructed here, but it's not a huge deal